### PR TITLE
Update Transactions.md

### PR DESCRIPTION
**Pull request by Celia18305**
Add description for signature scope

### CODE REVIEW

File: docs/n3/foundation/Transactions.md [hunk 1]
```diff
@@ -168,3 +154,258 @@ Here is an example of a JSON-format transaction, where the script and witnesses
...
+                  "expression": true
+              }
+          }
+      ]
+  }
+  ```
+
+- Not: Logical NOT. It reverses other conditions.
+
+  “expression”=<Condition>
```
Comment (apisit) [inline docs/n3/foundation/Transactions.md:214]: Unfortunately, We can't use <> in md file. You can either use another kind of parenthesis or use \ like the following
`\<bool\>`

File: docs/n3/foundation/Transactions.md [hunk 2]
```diff
@@ -168,3 +154,258 @@ Here is an example of a JSON-format transaction, where the script and witnesses
...
+
+Action(Allow|Deny) and Condition
+
+The execution logic is to execute the condition and, if is met, return Action, where Allow represents a successful check and Deny represents a failed check.
+
+#### WitnessCondition
+
+- Boolean: true|false
+
+  “expression” = <bool>
```
Comment (apisit) [inline docs/n3/foundation/Transactions.md:194]: I think the easiest way is to find and replace in this doc. 

Find: < and Replace: \<
Find: > and Replace: \>

**Review by apisit — CHANGES_REQUESTED** [review]
Thanks for updating. 

left a comment in there. So basically, md file doesn't allow < and >. You will have to find and replace the following.

`<` to `\<`
`>` to `\>`

so from `<bool>` it becomes `\<bool\>`

File: docs/n3/foundation/Transactions.md (hunk 2 above)
Comment (Celia18305) [reply docs/n3/foundation/Transactions.md]: I have done the replacement. Please kindly check.

**Pull request merged**